activiti-services/activiti-services-gateway/src/main/resources/application.properties

```diff
@@ -1,3 +1,23 @@
 server.port=8080
 spring.application.name=gateway
-management.security.enabled=false
\ No newline at end of file
+management.security.enabled=false
+
+spring.rabbitmq.host=rabbitmq
+spring.rabbitmq.port=5672
+eureka.client.serviceUrl.defaultZone=http://activiti-services-registry:8761/eureka/
+hystrix.command.default.execution.timeout.enabled=false
```

Comment:
I found I had to do this as it runs a bit slower in docker, at least on the first routing action. Otherwise it times out.